src/ApiCompat/Microsoft.DotNet.ApiCompat.Shared/ValidateAssemblies.cs

```diff
@@ -87,6 +88,23 @@ internal static class ValidateAssemblies
             // Execute the enqueued work item(s).
             apiCompatRunner.ExecuteWorkItems();
 
+            if (serviceProvider.CompatibilityLogger.SuppressionWasLogged)
+            {
+                if (!generateSuppressionFile)
+                {
+                    serviceProvider.CompatibilityLogger.LogMessage(
+                        MessageImportance.High,
+                        "Either fix breaking change, or update compatibility suppression files, rebuild with \\p:GenerateCompatibilitySuppressions or --generate-suppression-file.");
```

Comment:
Please localize the message (there's already a Resources.resx in the Shared project)